layers/+emacs/org/packages.el

```diff
@@ -122,11 +122,11 @@
     :defer t
     :init
     (progn
-      (defun spacemacs//org-babel-do-load-languages ()
-        "Load all the languages declared in `org-babel-load-languages'."
+      (define-advice org-babel-execute-src-block (:before (&rest _) ADV)
```

Comment:
it could be better if it has a more informative name than ADV

Reply:
Hi @lebensterben 
I had changed the suffix to "load-lang". It will be more clear. Thanks